host_cleanup.sh

```diff
@@ -64,3 +64,4 @@ sudo ebtables --flush
 
 # Kill any lingering proxy
 sudo pkill -f oc.*proxy
+sudo cp /etc/resolv.conf.bk /etc/resolv.conf
```

Comment:
Not sure why this is needed, perhaps left in inadvertently?

Reply:
ah, sorry. This is a workaround for a dev-scripts bug in my development machine, where it always nukes my dns config xD